### PR TITLE
Add MASTER_IPS and MASTER_INTERNAL_IPS variables

### LINKED ISSUE #14612: Update master-ip vars to plural
**What should be cleaned up or changed**:
Part 1 for https://github.com/kubernetes/perf-tests/issues/595
Multi-master support has been added to ClusterLoader2, update the variables in test-infra to plural.
**Provide any links for context**:
https://github.com/kubernetes/perf-tests/issues/595
https://github.com/kubernetes/perf-tests/pull/544

### PR DESCRIPTION
**Pull request by Jukie**
Fixes #14612 and is part 1 for https://github.com/kubernetes/perf-tests/issues/595

### CODE REVIEW

File: kubetest/e2e.go [hunk 1]
```diff
@@ -603,35 +603,46 @@ func kubemarkUp(dump string, o options, deploy deployer) error {
...
 		return err
 	}
+	// Leaving this temporarily until https://github.com/kubernetes/perf-tests/issues/595 is closed
 	// MASTER_IP variable is required by the clusterloader. It requires to have master ip provided,
 	// due to master being unregistered.
-	if err := os.Setenv("MASTER_IP", strings.TrimSpace(string(masterIP))); err != nil {
+	if err := os.Setenv("MASTER_IP", strings.TrimSpace(string(masterIPs))); err != nil {
+		return err
+	}
+	// MASTER_IPS variable is required by the clusterloader. It requires to have master ip(s) provided,
```
Comment (mm4tt) [inline kubetest/e2e.go]: I think this is no longer true in kubemark (masters not being registered). Given that, I think we can get rid of all this logic, as clusterloader2 should be able to obtain master info (including internal / public ips) from the node list.

Adding @mborsz, who implemented HA support in kubemark and will be able to confirm or correct me.

File: kubetest/e2e.go [hunk 2]
```diff
@@ -603,35 +603,46 @@ func kubemarkUp(dump string, o options, deploy deployer) error {
 		return err
 	}
 
-	masterIP, err := control.Output(exec.Command(
+	masterIPs, err := control.Output(exec.Command(
```
Comment (mm4tt) [inline kubetest/e2e.go]: This will return a single masterIp so changing it to plural doesn't make sense.

File: kubetest/e2e.go (hunk 1 above)
Comment (Jukie) [reply kubetest/e2e.go]: Alright if that's the case I'll remove it. Was I working on the wrong stuff you proposed in https://github.com/kubernetes/perf-tests/issues/595?

Comment (mm4tt) [reply kubetest/e2e.go]: That was my mistake, I wasn't aware that these envs are set only in one place and only for kubemark.
I think kubemark doesn't require them anymore, but let's wait for @mborsz to confirm.

Comment (mborsz) [reply kubetest/e2e.go]: Yes, the env variables here are no longer needed for kubemark (and any other cluster with master registered).

**Pull request closed without merging**